build.ps1

```diff
@@ -42,7 +42,25 @@ Trace-Log "Build #$BuildNumber started at $startTime"
 
 $BuildErrors = @()
     
-Invoke-BuildStep 'Getting private build tools' { Install-PrivateBuildTools } `
+Invoke-BuildStep 'Getting private build tools' { 
+        Install-PrivateBuildTools 
+
+        if (-Not (Test-Path $PrivateRoot)) {
+            return
+        }
+
+        # Copy over the configuration file from the private build tools.
+        $SourcesDirectory = $env:Build_SourcesDirectory
+        $configFileName = "$env:ConfigurationName.json"
```

Comment:
String formatting like this scares me in PowerShell. Could we do `"$(env:ConfigurationName) abc"` instead?

Reply:
Using `$($env:ConfigurationName)` because `$(env:ConfigurationName)` doesn't work